backends/bmv2/CMakeLists.txt

```diff
@@ -16,74 +16,84 @@
 # compiling for the simple_switch target.
 
 # sources for backend executable
-set (BMV2_SRCS bmv2.cpp)
-add_cpplint_files (${CMAKE_CURRENT_SOURCE_DIR} "${BMV2_SRCS}")
-
-set (BMV2_BACKEND_SRCS
-  JsonObjects.cpp
-  action.cpp
-  analyzer.cpp
-  backend.cpp
-  control.cpp
-  deparser.cpp
-  errorcode.cpp
-  expression.cpp
-  extern.cpp
-  extractArchInfo.cpp
-  globals.cpp
-  header.cpp
-  helpers.cpp
-  lower.cpp
-  metermap.cpp
-  midend.cpp
-  parser.cpp
-  portableSwitch.cpp
-  sharedActionSelectorCheck.cpp
-  simpleSwitch.cpp
+set (BMV2_SIMPLE_SWITCH_SRCS
+    simple_switch/main.cpp
+    simple_switch/simpleSwitch.cpp
+    simple_switch/simpleSwitch.h
+    )
+add_cpplint_files (${CMAKE_CURRENT_SOURCE_DIR} "${BMV2_SIMPLE_SWITCH_SRCS}")
+
+set (BMV2_PORTABLE_SWITCH_SRCS
+    portable_switch/main.cpp
```

Comment:
Same suggestion that @antoninbas had for bmv2, shall we call it psa here as well?